libs/tokens/src/utils/getTokenLogoUrls.ts

```diff
@@ -0,0 +1,25 @@
+import { cowprotocolTokenUrl, TokenWithLogo } from '@cowprotocol/common-const'
+import { uriToHttp } from '@cowprotocol/common-utils'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+
+export function getTokenLogoUrls(token: TokenWithLogo | undefined): string[] {
+  const fallbackUrls = token?.address
+    ? [
+        cowprotocolTokenUrl(token.address, token.chainId as SupportedChainId),
+        cowprotocolTokenUrl(token.address.toLowerCase(), token.chainId as SupportedChainId),
+        cowprotocolTokenUrl(token.address.toLowerCase(), SupportedChainId.MAINNET),
+      ]
```

Comment:
You mentioned 4 options in the description/diagrams where the last would be Trust.
Is it not here the place where it should be?

Reply:
I will add it soon.
https://github.com/cowprotocol/cowswap/pull/3201 - left it as TODO here